Formula/tcl-tk.rb

```diff
@@ -5,7 +5,7 @@ class TclTk < Formula
   mirror "https://fossies.org/linux/misc/tcl8.6.13-src.tar.gz"
   sha256 "43a1fae7412f61ff11de2cfd05d28cfc3a73762f354a417c62370a54e2caf066"
   license "TCL"
-  revision 3
+  revision 4
```

Comment:
Seems like this matters only at build-time, so we can probably drop the `revision` bump here.

Reply:
I made a new branch for the commit without changing the revision.

(If it's ultimately decided to bump the revision again that can be a new commit added by anyone.)

See: #133631